src/modules/PackageSlideShow.rb

```diff
@@ -873,70 +738,43 @@ def CdStatisticsTableItems
       Builtins.foreach(@remaining_sizes_per_cd_per_src) do |inst_src|
         Builtins.y2milestone("src #%1: %2", src_no, inst_src)
         # Ignore repositories from where there is nothing is to install
-        if Ops.greater_than(ListSum(inst_src), 0)
-          # Add heading for this repository
-          itemList = Builtins.add(
-            itemList,
-            SlideShow.TableItem(
-              Builtins.sformat("src(%1)", src_no),
-              Ops.get(@inst_src_names, src_no, ""),
-              "",
-              "",
-              ""
+        next if ListSum(inst_src) < 1
+
+        cd_no = 0
+
+        Builtins.foreach(inst_src) do |src_remaining|
```

Comment:
Ruby style instead of Builtins.... ? :-)

Reply:
as it should be guranteed that inst_src is non-nil, I will change it.